addons/api/package.json

```diff
@@ -10,67 +10,70 @@
   "scripts": {
     "build:development": "ember build",
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
```

Comment:
None blocking comment: I guess `ember-cli` deprecated `npm-run-all` and installed `concurrently`? or we did so?

Reply:
Yeah this was all `ember-cli` 😆